src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$oneBlockMargin: 101px;
```

Comment:
let's create another variable $gap: 1px and separate it from $blockWidth: 100px;